update_version

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+
+cd "$(dirname "$0")"
+set -e
+
+[[ -n "$1" ]] || { echo "Usage: $0 <version>"; exit 1; }
+
+for entry in `find . -mindepth 2 -name Cargo.toml -print`; do
+    echo $entry
+    line=$(grep -n -m1 "version" $entry | cut -d: -f2)
+    CURRENT=$(echo ${line} | awk -F'"' '{print $2}')
+    sed -i "" "s/$CURRENT/$1/g" $entry
```

Comment:
I think that you need a -e here (and the `-i ""` might fail on mac, I can only remember that `-i~` works everywhere.

Reply:
This works om my mac.

Reply:
-i "" fails on linux for me. I suggest removing the space between `-i` and `""` changing it to

```
sed -i"" "s/$CURRENT/$1/g" $entry
``` 

Which only doesn't works on MacOS up to 10.8[[1]](https://stackoverflow.com/a/4247319), which had end of support around September 2016[[2]](https://en.wikipedia.org/wiki/MacOS_version_history#Releases). So it works on all platforms I think we care about.

Reply:
OK, good.  Note that `sed -i""` is exactly equivalent to just `sed -i`.

Reply:
Yes, on linux it is equivalent. On macos only `sed -i` doesn't work if I understand it correctly.